setupbase.py

```diff
@@ -135,7 +135,6 @@ def find_package_data():
         pjoin(components, "bootstrap", "js", "bootstrap.min.js"),
         pjoin(components, "bootstrap-tour", "build", "css", "bootstrap-tour.min.css"),
         pjoin(components, "bootstrap-tour", "build", "js", "bootstrap-tour.min.js"),
-        pjoin(components, "es6-promise", "*.js"),
```

Comment:
It's still failing and I think it's because it's looking for these files before `npm install`...